lib/std/fmt.zig

```diff
@@ -456,6 +456,18 @@ pub fn formatType(
                 }
                 if (ptr_info.child == u8) {
                     return formatText(value, fmt, options, writer);
+                } else if (fmt.len > 0 and ((fmt[0] == 'v') or (fmt[0] == 'V'))) {
```

Comment:
`v` for vendetta? Why bother with two format specifiers when you're treating them the same way?
This way you can't specify what format to use for the content, I'd accept two-char formats too (eg. `xv` or `vx` to print the slice contents as hex).

Maybe this is better suited for the `*` specifier, that's supposed to deref the input.